src/main/java/frc/robot/Robot.java

```diff
@@ -29,12 +29,11 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-  public static final boolean isCompetition = true;
+  public static final boolean isCompetition = false;
```

Comment:
in Main, we should isCompetiton set to True to remove any chance of mistake at competition time